psiminer-core/src/main/kotlin/storage/raw/RawStorage.kt

```diff
@@ -0,0 +1,30 @@
+package storage.raw
+
+import Dataset
+import com.intellij.psi.PsiElement
+import com.intellij.psi.impl.source.tree.LeafPsiElement
+import labelextractor.LabeledTree
+import psi.nodeProperties.isHidden
+import psi.nodeProperties.technicalToken
+import storage.Storage
+import java.io.File
+
+class RawStorage(outputDirectory: File) : Storage(outputDirectory) {
+    override val fileExtension: String = "txt"
+
+    private fun processElement(element: PsiElement): String {
+        if (element.technicalToken != null) return element.technicalToken!!
```

Comment:
Usage of `!!` should be avoided when possible. Store token in a variable or use elvis operator.